application/src/main/java/com/softserve/lv460/application/controller/ActionController.java

```diff
@@ -0,0 +1,71 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.constant.HttpStatuses;
+import com.softserve.lv460.application.dto.action.ActionDTO;
+import com.softserve.lv460.application.mapper.action.ActionMapper;
+import com.softserve.lv460.application.service.ActionService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/actions")
+@CrossOrigin
+public class ActionController {
+
+  private ActionService actionService;
+  private ActionMapper mapper;
+
+  @ApiOperation(value = "Create new action")
+  @ApiResponses(value = {
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = ActionDTO.class),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+  })
+  @PostMapping
+  public ResponseEntity<ActionDTO> create(@RequestBody ActionDTO dto) {
+    return ResponseEntity.status(HttpStatus.CREATED).body(mapper.toDto(actionService.create(mapper.toEntity(dto))));
+  }
+
+  @ApiOperation(value = "Return list of action")
+  @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK, response = ActionDTO.class),
+        @ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
```

Comment:
404 is not applicable here. When there are not actions - you just return empty list